prow/cluster/jobs/istio/proxy/istio.proxy.master.yaml

```diff
@@ -124,5 +124,9 @@ postsubmits:
           limits:
             memory: "60Gi"
             cpu: "16000m"
+      volumes:
```

Comment:
do we need a volumeMount?

Reply:
Also doesn't `preset-service-account: "true"` do this already?

Reply:
I am trying to figure out how to make the test be happy :|

Reply:
https://github.com/istio/istio/issues/15714#issuecomment-517483833

Reply:
We have plenty of other jobs that push to GCR though.. every e2e test in istio/istio does and just ` preset-service-account: "true"` is sufficient. The ` preset-service-account: "true"` preset looks like it already adds this volume actually?